src/components/screens/sprint-view-screen.tsx

```diff
@@ -156,8 +157,7 @@ const SprintViewScreen = () => {
           {resourceAllocations?.map((resourceAllocations) => (
             <TaskTable
               key={resourceAllocations.phase?.severaPhaseId}
```

Comment:
Is there any reason not to pass the resourceAllocation.project.severaProjectId here? Not a big deal but as we don't seem to be passing the phase here but the project it maybe is a bit clearer?